src/tasks/runstart.ts

```diff
@@ -123,13 +132,21 @@ export const RunStartQuest: Quest = {
       limit: { tries: 1 },
     },
     {
-      name: "Get Codpiece",
+      name: "Get Floundry item",
       completed: () => get("_floundryItemCreated") || get("instant_saveFloundry", false),
       do: (): void => {
-        retrieveItem($item`codpiece`, 1);
-        use($item`codpiece`, 1);
-        create($item`oil cap`, 1);
-        autosell($item`oil cap`, 1);
+        if (myPrimestat() === $stat`Muscle`) {
+          retrieveItem($item`fish hatchet`);
+        } else if (myPrimestat() === $stat`Mysticality`) {
+          retrieveItem($item`codpiece`, 1);
+          use($item`codpiece`, 1);
+          create($item`oil cap`, 1);
+          autosell($item`oil cap`, 1);
+        } else if (myPrimestat() === $stat`Moxie`) {
+          retrieveItem($item`bass clarinet`);
+          use($item`bass clarinet`, 1);
+          autosell($item`white pixel`, 10);
+        }
```

Comment:
How will this potentially affect meat sources for low shinies? I'm also assuming there are no significant conflict of NC% slots for the NC test

Reply:
Also, if we aren't defaulting to always picking codpiece, the default outfit should probably be updated to not use the codpiece by default if we aren't a myst class